cmd/nodecmd/validateSubnet.go

```diff
@@ -127,28 +135,20 @@ func waitForNodeToBePrimaryNetworkValidator(nodeID ids.NodeID) error {
 
 func validateSubnet(_ *cobra.Command, args []string) error {
 	clusterName := args[0]
-	subnetName := args[1]
+	subnetName = args[1]
```

Comment:
I rather prefer the option of not using the global variable, but instead passing this as argument, as in another PR.
in any case let's the other PR implementation prevail. I prefer no global as there is no need to use global here (there is no such flag)